script/build-remote

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+. $(dirname $0)/build-common
+
+BUILD_CONTAINER=$(docker run -d docker-machine gox "${OS_PLATFORM_ARG[@]}" "${OS_ARCH_ARG[@]}" -output="docker-machine_{{.OS}}-{{.Arch}}" -ldflags="-w -X github.com/docker/machine/version.GitCommit `git rev-parse --short HEAD`")
+cleanup_container() {
+    docker rm -v ${BUILD_CONTAINER}
+}
+trap cleanup_container EXIT
+
+docker logs -f ${BUILD_CONTAINER} &
+BUILD_STATUS=$(docker wait ${BUILD_CONTAINER})
+if [[ ${BUILD_STATUS} != 0 ]]; then exit ${BUILD_STATUS}; fi
+
+BUILT_IMAGE=$(docker commit ${BUILD_CONTAINER})
+cleanup_image() {
+    cleanup_container
+    docker rmi ${BUILT_IMAGE}
+}
+trap cleanup_image EXIT
+
+echo "Copying built binaries to ./:"
+for f in `docker run --rm ${BUILT_IMAGE} ls | grep 'docker-machine_'`; do
```

Comment:
I'm personally always scared when I see the output of `ls` scripted.  In this case, it's _probably_ safe, but the output from `ls` is usually supposed to be human-readable, so I'd trust a raw shell glob more personally, something like:
```bash
for f in $(docker run --rm ${BUILT_IMAGE} sh -c 'echo docker-machine_*'); do
    ...
```